app/src/components/searchWidget/styles/widget.scss

```diff
@@ -10,6 +10,7 @@ $break-small: 375px;
     width: 455px;
     margin: 20px;
     text-align: left;
+    z-index: 999999;
```

Comment:
Please rename related class.
Noone knows what is '.pickup-location'

Reply:
I did't create that class, it was there before

Reply:
So not sure if I should change it or not .